### ISSUE TITLE
False positive SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse when const used in named argument

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1215]
```php
<?php declare(strict_types = 1);

namespace Test;

use function str_pad;

use const STR_PAD_RIGHT;

echo str_pad('123', 1, pad_type: STR_PAD_RIGHT);
```

```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <file>./packages</file>
    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses"/>
</ruleset>
```

```
./vendor/bin/phpcs ./packages/Bug.php

FILE: /project/packages/Bug.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 7 | ERROR | [x] Type STR_PAD_RIGHT is not used in this file.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 188ms; Memory: 10MB
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/a9af14f9f8538d508f5fbee1b48a757355ce5b00
